components/Buttons/src/ColorThemer/MDCOutlinedButtonColorThemer.h

```diff
@@ -22,10 +22,9 @@
  `MDCButton`'s `-applyOutlinedThemeWithScheme:`
  Learn more at docs/theming.md#migration-guide-themers-to-theming-extensions
  */
-@interface MDCOutlinedButtonColorThemer : NSObject
-@end
-
-@interface MDCOutlinedButtonColorThemer (ToBeDeprecated)
+__deprecated_msg("Please use [MDCButton applyOutlinedThemeWithScheme:] instead. (Note: Color "
```

Comment:
Tip: Consider suppressing deprecation warnings in the project's use of this API.

Reply:
I prefer we get these warning and start fixing them - just like our clients...